.github/workflows/main.yml

```diff
@@ -215,6 +215,7 @@ jobs:
     - name: Shader Validation Tests
       if: matrix.test_shaders == 'ON' && runner.os == 'Windows'
       run: |
+        vcpkg/vcpkg remove glslang
```

Comment:
Now that the glslang test is disabled, I believe we can remove this workaround from recent pull requests.